src/utilities/__tests__/findBreakingChanges-test.js

```diff
@@ -1056,215 +714,116 @@ describe('findBreakingChanges', () => {
-        field1: { type: GraphQLString },
-        field2: { type: GraphQLString },
-      },
-    });
-    const typeThatHasBreakingFieldChangesNew = new GraphQLInterfaceType({
-      name: 'TypeThatHasBreakingFieldChanges',
-      fields: {
-        field2: { type: GraphQLBoolean },
-      },
-    });
-
-    const typeInUnion1 = new GraphQLObjectType({
-      name: 'TypeInUnion1',
-      fields: {
-        field1: { type: GraphQLString },
-      },
-    });
-    const typeInUnion2 = new GraphQLObjectType({
-      name: 'TypeInUnion2',
-      fields: {
-        field1: { type: GraphQLString },
-      },
-    });
-    const unionTypeThatLosesATypeOld = new GraphQLUnionType({
-      name: 'UnionTypeThatLosesAType',
-      types: [typeInUnion1, typeInUnion2],
-    });
-    const unionTypeThatLosesATypeNew = new GraphQLUnionType({
-      name: 'UnionTypeThatLosesAType',
-      types: [typeInUnion1],
-    });
-
-    const enumTypeThatLosesAValueOld = new GraphQLEnumType({
-      name: 'EnumTypeThatLosesAValue',
-      values: {
-        VALUE0: { value: 0 },
-        VALUE1: { value: 1 },
-        VALUE2: { value: 2 },
-      },
-    });
-    const enumTypeThatLosesAValueNew = new GraphQLEnumType({
-      name: 'EnumTypeThatLosesAValue',
-      values: {
-        VALUE1: { value: 1 },
-        VALUE2: { value: 2 },
-      },
-    });
-
-    const interface1 = new GraphQLInterfaceType({
-      name: 'Interface1',
-      fields: {
-        field1: { type: GraphQLString },
-      },
-    });
-
-    const typeThatLosesInterfaceOld = new GraphQLObjectType({
-      name: 'TypeThatGainsInterface1',
-      interfaces: [interface1],
-      fields: {
-        field1: {
-          type: GraphQLString,
-        },
-      },
-    });
-
-    const typeThaLosesInterfaceNew = new GraphQLObjectType({
-      name: 'TypeThatGainsInterface1',
-      interfaces: [],
-      fields: {
-        field1: {
-          type: GraphQLString,
-        },
-      },
-    });
-
-    const directiveThatIsRemoved = GraphQLSkipDirective;
-    const directiveThatRemovesArgOld = new GraphQLDirective({
-      name: 'DirectiveThatRemovesArg',
-      locations: [DirectiveLocation.FIELD_DEFINITION],
-      args: {
-        arg1: {
-          name: 'arg1',
-        },
-      },
-    });
-    const directiveThatRemovesArgNew = new GraphQLDirective({
-      name: 'DirectiveThatRemovesArg',
-      locations: [DirectiveLocation.FIELD_DEFINITION],
-    });
-    const nonNullDirectiveAddedOld = new GraphQLDirective({
-      name: 'NonNullDirectiveAdded',
-      locations: [DirectiveLocation.FIELD_DEFINITION],
-    });
-    const nonNullDirectiveAddedNew = new GraphQLDirective({
-      name: 'NonNullDirectiveAdded',
-      locations: [DirectiveLocation.FIELD_DEFINITION],
-      args: {
-        arg1: {
-          name: 'arg1',
-          type: GraphQLNonNull(GraphQLBoolean),
-        },
-      },
-    });
-    const directiveRemovedLocationOld = new GraphQLDirective({
-      name: 'Directive Name',
```

Comment:
This directive had incorrect name, according to the spec, spaces should not be allowed on directive name:
https://facebook.github.io/graphql/draft/#DirectiveDefinition
https://facebook.github.io/graphql/draft/#Name

Reply:
👍